Vagrantfile

```diff
@@ -12,8 +12,22 @@ ln -s /opt/rpm/SPECS/vault.spec /home/vagrant/rpmbuild/SPECS/
 find /opt/rpm/SOURCES -type f -exec ln -s {} rpmbuild/SOURCES \\\;
```

Comment:
this is good for local testing but we usually want just .spec files in these repos